### DISCUSSION TITLE
Does this work for command-line apps?

### DISCUSSION THREAD

**Discussion by cabintech** [saphirepankaj/jexec-build#5] [Q&A] [answered]
Should this work for console applications? I wrote a simple 1-line main:

	public static void main(String[] args) {
		System.out.println("Got here!");

	}

I built a jar, ran this tool and unpacked the output zip file. 
I opened a command window, navigated to the unpacked directory and ran "JExec.jar". 
It displayed a spinning circle graphic forever and CPU usage was very high. I had to kill the Java process.

**Comment by saphirepankaj** [comment, marked as answer]
Yes, it works for java console application as well, we have done some major improvements in our solution, download the latest file and try again. Watch our you video to know more, https://youtu.be/eb0_UWaa8cU
